embulk-core/src/main/java/org/embulk/deps/SelfContainedJarFile.java

```diff
@@ -55,7 +55,7 @@
             return;
         }
 
-        final JarInputStream jarInputStream;
+        JarInputStream jarInputStream = null;
```

Comment:
Did you have special reasons why we needed to remove `final` and assign `null` here?

Reply:
Not needed if the next issue you raised is addressed. thanks.